.github/workflows/golang.yaml

```
@@ -15,17 +15,7 @@
 name: Golang
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
-  push:
-    branches:
-      - main
-      - release-*
+  workflow_call:
```

Given that we want these to run on PRs so that they trigger for dependabot we may want to update this to:
```suggestion
  workflow_call: {}
  pull_request:
    types:
      - opened
      - synchronize
    branches:
      - main
      - release-*
```